packages/relay-compiler/generic/codegen/RelayCodegenRunner.js

```diff
@@ -36,7 +36,10 @@ export type ParserConfig = {|
   getFileFilter?: (baseDir: string) => FileFilter,
   getParser: (baseDir: string) => FileParser,
   getSchema: () => GraphQLSchema,
-  watchmanExpression: WatchmanExpression,
+  extensions: Array<string>,
+  include: Array<string>,
+  exclude: Array<string>,
+  watchman: boolean,
```

Comment:
I was working on merging this, but unfortunately this breaking change will prevent me from doing so.

This `RelayCodegenRunner` is public API, and it's used extensively by Relay projects internally at Facebook which provide complex `watchmanExpression` that cannot be replicated simply with include/exclude.